docs/solutions/k8s-on-dcos.md

```diff
@@ -0,0 +1,271 @@
+# Blueprint - Kubernetes on the Mesosphere DCOS
+
+## Contents
+
+1. [Terminology](#1-terminology)
+1. [Use Cases](#2-use-cases)
+ 1. Case study 1: Financial Industry
+ 1. Case study 2: Gaming
+1. [Solution](#3-solution)
+1. [Components](#4-components)
+ 1. Prerequisites
+ 1. Bill of Materials
+1. [Example Deployment](#5-example-deployment)
+1. [Resources](#6-resources)
+
+[Kubernetes on DCOS](http://mesosphere.com/kubernetes) adds Kubernetes to Mesosphere’s Datacenter Operating System (DCOS) in order to help enterprises unlock the next generation of datacenter scale, efficiency and automation. [Kubernetes](http://kubernetes.io/) and the Mesosphere [DCOS](http://mesosphere.com/learn/) together provide a powerful platform for building and deploying microservice-based applications and operate containerized (e.g., using Docker) applications in a scalable, elastic and secure manner. The Mesosphere DCOS, which is based on Apache Mesos, is integrating Kubernetes as a standard component which will be made available as part of the Mesosphere DCOS Community Edition (free) and the Mesosphere DCOS Enterprise Edition (paid).
+
+In this document, you'll learn how to use Kubernetes on the Mesosphere DCOS in order to build scalable, resilient and secure application architectures. You'll see how these principles apply to real-world scenarios through an example deployment.
+
+## 1. Terminology
+
+**Docker**, [docker.com](https://www.docker.com/): is a popular container solution that enables deployable apps to be quickly assembled from components and eliminates the friction between development, QA and production environments. It consists of 1. the Docker Engine, a portable, lightweight runtime and packaging tool, and 2. Docker Hub, a cloud service for sharing applications and automating workflows. For on-premise deployments, a private registry can be used to serve the Docker images.
+
+**Kubernetes**, [kubernetes.io](http://kubernetes.io/): is an open source orchestration system for Docker containers. It actively manages workloads to ensure that their state matches the users declared intentions. Kubernetes provides declarative primitives—such as pods, labels and services—to group the Docker containers that make up an application into logical units, for easy management and discovery.
```

Comment:
Taking the best parts of your version and the original version from kubernetes.io, how about: Kubernetes is an open source orchestration system for Docker containers. It schedules containers onto nodes in a compute cluster and actively manages workloads to ensure that their state matches the user's declared intentions. Kubernetes provides declarative primitives—such as pods, labels and services—to group the Docker containers that make up an application into logical units, for easy management and discovery."